documentation/rewrite.asciidoc

```diff
@@ -0,0 +1,35 @@
+:toc:
+toc::[]
+
+= rewrite
+
+The rewrite commandlet allows the use of https://docs.openrewrite.org/[Openrewrite], a Maven based tool that can be used to refactor code with recipes for multiple languages and tools. Before using this tool, we recommend that you read the https://docs.openrewrite.org/running-recipes/getting-started[Getting-Started] page.
+
+== Usage
+The commandlet excepts the name of a recipe such as
```

Comment:
```suggestion
The commandlet accepts the name of a recipe such as
```

Reply:
@timtebeek thanks for your feedback to improve. I merged roughly at the same moment.
However, I just included your suggestion so it is not lost.